transients_nearby_galaxies/transients_nearby_galaxies.ipynb

```diff
@@ -0,0 +1,1182 @@
+{
```

Comment:
<p>Could we break down</p><pre class="ql-syntax" spellcheck="false">tmin=t_min=[row['minStart'][0],row['maxStart'][0]]
</pre><p> </p><p>Maybe we could do something like</p><pre class="ql-syntax" spellcheck="false">minStart = row['minStart'][0]
maxStart = row['maxStart'][0]
obs_table = ..... t_min=[minStart, maxStart]
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>